src/stringobj.c

```diff
@@ -1906,26 +1906,16 @@ Obj FuncSplitString (
     return FuncSplitString( self, string, seps, wspace );
   }
   
-  /* reset SPLITSTRINGSEPS (in case of previous error) */
-  if (SPLITSTRINGSEPS[256] != 0) {
-    for(i=0; i<257; i++) SPLITSTRINGSEPS[i] = 0;
-  }
-  
   /* set SPLITSTRINGSEPS by setting positions of characters in rem to 1 */
   len = GET_LEN_STRING(seps);
   s = CHARS_STRING(seps);
-  SPLITSTRINGSEPS[256] = 1;
```

Comment:
I have no addressed this, and also removed the changes touching `stringobj.h`, so as to not conflict with your work on that.